spec/models/books_model_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
+    expect(:genre).to be_present
+  end
+  it do
+    expect(:author).to be_present
+  end
+  it do
+    expect(:image).to be_present
+  end
+  it do
+    expect(:title).to be_present
+  end
+  it do
+    expect(:publisher).to be_present
+  end
+  it do
+    expect(:year).to be_present
+  end
+
+  subject(:book) { build(:book) }
+
+  it do
+    is_expected.to be_valid
+  end
+
+  describe '#create' do
```

Comment:
Congrats for making specs for each case of the validation! I know the card told you to add this spec, but I want to explain you why they are not really needed.

If you look at the spec you have just added you are testing that the `create` method is validating your model. This method is defined by rails. You are like testing rails, and rails is already tested by them. So in this case you should only test that you have the right validations defined, as you wrote before. Then you assume that rails works like it says and it will call your validations.

If I was not clear, please let me know and I'll try to explain it again